annotation-file-utilities/build.gradle

```diff
@@ -76,6 +76,14 @@ task skinnyJar(type: ShadowJar, dependsOn: compileJava) {
   include 'org/checkerframework/afu/annotator/**/*.class'
 }
 
+createCheckTypeTask(project.name, 'CompilerMessages',
```

Comment:
@smillst Is it possible to define this once in `./build.gradle` rather than 3 times (in framework, checker, and annotation-file-utilities)?